.github/workflows/scripts/validate-coderabbit.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+echo "Step 1: Fetching all PR reviews with pagination..."
+
+page=1
+all_reviews="[]"
+
+while true; do
+  echo "Fetching page $page..."
+  response=$(curl -s -f -H "Authorization: token $GITHUB_TOKEN" \
+    "https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls/${PR_NUMBER}/reviews?per_page=100&page=$page") || {
```

Comment:
Should this be a parameter passed to the script?
Looking at the code the script may not be appropriate. It relies on too many GitHub specific values

Reply:
yes it needs pr number and repo name